.github/workflows/ci.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   pull_request:
-    branches: [ main ]
+    branches: [ main, 'releases/**']
```

Comment:
The meaning of this is to run CI on PRs targeting any `releases/**` branch. What about removing line 5 and run it on any PR? It's just CI not CD.

Reply:
I agree with @roschaefer ... that is what I'd like.

Reply:
done with: eca70b8a0a4d87a978e408db2a3da0b3e645646a